### PR TITLE
Add message to show only added features and always show full list of …

### PR DESCRIPTION
**Pull request by brenthdaniel**
…installed features

Implementation of #20372

### CODE REVIEW

**Comment by brenthdaniel** [comment]
#pb

**Comment by brenthdaniel** [comment]
#build

File: dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops [hunk 1]
```diff
@@ -345,3 +345,13 @@ tool.userfeature.install.location.invalid=CWWKF1036E: The {0} user feature canno
 tool.userfeature.install.location.invalid.explanation= The user feature is not a core feature therefore cannot be installed to core.
 tool.userfeature.install.location.invalid.useraction= Install the user feature to usr. 
 
+FEATURES_ADDED_DELTA=CWWKF1037I: The server added features {0} to the existing feature set.
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops]: if {0}  is a list of added features:

```suggestion
FEATURES_ADDED_DELTA=CWWKF1037I: The server added the {0} features to the existing feature set.
```

File: dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops [hunk 2]
```diff
@@ -345,3 +345,13 @@ tool.userfeature.install.location.invalid=CWWKF1036E: The {0} user feature canno
...
+FEATURES_ADDED_DELTA.explanation=The server has installed a set of new features in addition to previously installed features.
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops]:
```suggestion
FEATURES_ADDED_DELTA.explanation=The server installed a set of new features in addition to previously installed features.
```

File: dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops [hunk 3]
```diff
@@ -345,3 +345,13 @@ tool.userfeature.install.location.invalid=CWWKF1036E: The {0} user feature canno
...
+FEATURES_ADDED_DELTA.useraction=No action is required.
+
+FEATURES_ADDED_CLIENT_DELTA=CWWKF1038I: The client added features {0} to the existing feature set.
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops]:
```suggestion
FEATURES_ADDED_CLIENT_DELTA=CWWKF1038I: The client added the {0} features to the existing feature set. 
```

File: dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops [hunk 4]
```diff
@@ -345,3 +345,13 @@ tool.userfeature.install.location.invalid=CWWKF1036E: The {0} user feature canno
 tool.userfeature.install.location.invalid.explanation= The user feature is not a core feature therefore cannot be installed to core.
 tool.userfeature.install.location.invalid.useraction= Install the user feature to usr. 
 
+FEATURES_ADDED_DELTA=CWWKF1037I: The server added features {0} to the existing feature set.
+FEATURES_ADDED_DELTA.explanation=The server has installed a set of new features in addition to previously installed features.
+FEATURES_ADDED_DELTA.useraction=No action is required.
+
+FEATURES_ADDED_CLIENT_DELTA=CWWKF1038I: The client added features {0} to the existing feature set. 
+FEATURES_ADDED_CLIENT_DELTA.explanation=The client has installed a set of new features in addition to previously installed features.
```
Comment (dmuelle) [inline dev/com.ibm.ws.kernel.feature.core/resources/com/ibm/ws/kernel/feature/internal/resources/ProvisionerMessages.nlsprops]:
```suggestion
FEATURES_ADDED_CLIENT_DELTA.explanation=The client installed a set of new features in addition to previously installed features.
```

**Review by dmuelle — CHANGES_REQUESTED** [review]
ID review, lmk if you have any questions, thanks

**Comment by brenthdaniel** [comment]
#build

**Review by dmuelle — DISMISSED** [review]
LGTM, thanks

**Comment by posmith** [comment]
L2 message review completed

**Comment by brenthdaniel** [comment]
#build

**Review by hlhoots — DISMISSED** [review]
LGTM

**Review by jimblye — DISMISSED** [review]

**Comment by brenthdaniel** [comment]
#build

**Review by hlhoots — APPROVED** [review]
LGTM

**Review by jimblye — APPROVED** [review]

**Comment by brenthdaniel** [comment]
#run-libby-bot

**Pull request merged**